community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/pipes/CountStoreRelationshipAggregationPipe.scala

```diff
@@ -0,0 +1,53 @@
+package org.neo4j.cypher.internal.compiler.v2_3.pipes
+
+import org.neo4j.cypher.internal.compiler.v2_3.ExecutionContext
+import org.neo4j.cypher.internal.compiler.v2_3.executionplan.Effects
+import org.neo4j.cypher.internal.compiler.v2_3.planDescription.{NoChildren, PlanDescriptionImpl}
+import org.neo4j.cypher.internal.compiler.v2_3.symbols.SymbolTable
+import org.neo4j.cypher.internal.frontend.v2_3.NameId
+import org.neo4j.cypher.internal.frontend.v2_3.ast.RelTypeName
+import org.neo4j.cypher.internal.frontend.v2_3.symbols._
+
+case class CountStoreRelationshipAggregationPipe(ident: String, startLabelName: Option[String],
```

Comment:
Same here we don't need to lookup the id for each row. Use LazyLabel or at least store the id as a field.